core/src/components/button/button.ionic.scss

```
@@ -11,12 +11,11 @@
   --focus-ring-color: #{globals.$ion-border-focus-default};
   --focus-ring-width: #{globals.$ion-border-size-050};
 
-  position: relative;
+  @include globals.typography(globals.$ion-body-action-md);
```

```suggestion
  @include globals.typography(globals.$ion-body-action-lg);
```

Is this right? I am seeing `body/action/lg` on the medium button. If you inspect Figma the comment doesn't match the label under the "Typography" section.